src/reportWebVitals.js

```diff
@@ -0,0 +1,13 @@
+const reportWebVitals = onPerfEntry => {
```

Comment:
This task does not need any reporting. Please remove this `reportWebVitals.js` file